site/playroom.config.js

```diff
@@ -18,6 +18,13 @@ module.exports = {
   openBrowser: false,
   port: 8082,
   webpackConfig: () => ({
+    module: {
+      rules: [
+        // Playroom has its own CSS loaders and sku has its own CSS loaders.
+        // We want to override Playroom's loaders and only use sku's ones.
+        { test: /\.css$/, use: [] },
+      ],
+    },
```

Comment:
Happy to apply this override for now, and look at possible alternative approaches to handling CSS at the Playroom end to not have to do this.